lib/active_merchant/billing/gateways/credorax.rb

```diff
@@ -253,10 +254,27 @@ def add_3ds_2_optional_fields(post, options)
 
       private
 
+      def isk_change_check(money)
+        if test?
```

Comment:
These timestamps look good and explicitly set it to UTC 👍

Reply:
Do you think Esmit's suggestion for freezing them (to the integer form) would be preferred?